docs/source/compile/guards-overview.rst

```diff
@@ -96,8 +96,8 @@ The function we have installed is ``convert_frame`` or
 over that nuance for now, let’s take a look at ``convert_frame_assert``,
 as ``convert_frame`` proxies to it.
 
-We can find it on `line 20 of convert_frame.py
-<https://github.com/pytorch/torchdynamo/blob/main/torchdynamo/convert_frame.py#L200>`__,
+We can find it on `line 222 of convert_frame.py
+<https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/convert_frame.py#L222>`__,
```

Comment:
It might be more sustainable to switch this with a permalink?

Reply:
Agree